packages/web3-eth/CHANGELOG.md

```diff
@@ -98,3 +98,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 -   Added `createAccessList` functionality ( #5780 )
+-   Added support of `safe` and `finalized` block tags (#5823)
```

Comment:
For `web3-eth` and `web3-rpc-methods` there wasn't any code changes to add support for the new tags, they just inherited support from other packages that were updated

Reply:
@spacesailor24 
I think the whole idea of the CHANGELOG.md is to inform the user about any change in the usage experience. And for the user, it does not make any difference if these changes came from updating some code in the package, or if the changes came from updating a dependency. So, if there was any user-experience change that is introduced in a specific version, then those must be mentioned in the CHANGELOG.md. 

In short, from the user perspective, before this update, it was not possible to pass `safe` or `finalized` block tags. And this change in behavior is introduced in this specific version. And this has to be documented here.